llama_stack/providers/remote/vector_io/chroma/chroma.py

```diff
@@ -85,6 +85,14 @@ async def query(self, embedding: NDArray, k: int, score_threshold: float) -> Que
     async def delete(self):
         await maybe_await(self.client.delete_collection(self.collection.name))
 
+    async def query_keyword(
+        self,
+        query_string: str | None,
```

Comment:
```suggestion
        query_string: Optional[str],
```

Reply:
Same goes here. Query string would be a required parameter for `query_keyword`. If keyword is not supported providers can give an notImplemented error.

Reply:
Updated.